src/olympia/blocklist/views.py

```diff
@@ -111,15 +125,25 @@ def get_items(apiver, app, appver=None):
     return items, details
 
 
-def get_plugins(apiver, app, appver=None):
+def get_plugins(apiver=3, app=None, appver=None):
     # API versions < 3 ignore targetApplication entries for plugins so only
     # block the plugin if the appver is within the block range.
+
+    app_query = (Q(app__isnull=True) |
```

Comment:
The `if app:`/`else:` is slightly better IMHO because it avoids manipulating `Q` objects before throwing them away.